projects/swimlane/ngx-ui/src/lib/components/json-editor/node-types/object-node.component.ts

```diff
@@ -145,14 +146,26 @@ export class ObjectNode implements OnInit, OnChanges {
    * Adds a new property to the model
    */
   addProperty(dataType: JsonSchemaDataType): void {
-    const propName = `${dataType.name} ${this.propertyCounter}`;
-    this.propertyCounter++;
-    const schema = JSON.parse(JSON.stringify(dataType.schema));
+    let propertyCounter = 1;
```

Comment:
This looks delicate. I would suggest to make sure we test both the schemaBuilderMode and the regular json editor. This function is kinda crucial.